layers/+lang/python/packages.el

```diff
@@ -63,6 +64,13 @@
     ;; packages for Microsoft's pyright language server
     (lsp-pyright :requires lsp-mode :toggle (eq python-lsp-server 'pyright))))
 
+(defun python/init-pet ()
+  (use-package pet
+    :hook (python-base-mode . pet-mode)
+    :if (and (executable-find "dasel")
+             (executable-find "sqlite3"))
```

Comment:
sqlite3 is not required unless you are running emacs < 29 or hand compiled a emacs without it, AFAIK, most emacs distributions have it. You may want to do similar [feature detection](https://github.com/wyuenho/emacs-pet/blob/92da4fa6ffd97fadb24c6fc5d46976606a0264c4/pet.el#L472-L473) before making it a requirement.

Reply:
Good point :), I will have a look